models/oci/error.go

```diff
@@ -48,3 +55,27 @@ func ErrGettingImage(err error) error {
 func ErrValidatingImage(err error) error {
 	return errors.New(ErrValidatingImageCode, errors.Alert, []string{"validating image failed"}, []string{err.Error()}, []string{"failed to validate the image"}, []string{"Try using a different image", "check if image is not malformed"})
 }
+
+func ErrConnectingToRegistry(err error) error {
+	return errors.New(ErrConnectingToRegistryCode, errors.Alert, []string{"connecting to registry failed"}, []string{err.Error()}, []string{"failed to connect to the registry"}, []string{"Try using a different registry", "check if registry URL is correct"})
+}
+
+func ErrFileNotFound(err error) error {
+	return errors.New(ErrFileNotFoundCode, errors.Alert, []string{"file not found"}, []string{err.Error()}, []string{"file not found"}, []string{"Try using a different file", "check if file exists"})
```

Comment:
Try using a different file. I see this `cause` at other errors as well, please update those.
For ErrFileNotFound, how's check correct filePath is provided and also including the filepath provided by the user.

Reply:
@RipulHandoo

Reply:
Addressed.